collections/src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -266,6 +284,19 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
     */
   def to[C1](factory: Factory[A, C1]): C1 = factory.fromSpecific(toIterable)
 
+  def toList: immutable.List[A] = immutable.List.from(toIterable)
+
+  def toVector: immutable.Vector[A] = immutable.Vector.from(toIterable)
+
+  def toMap[T, U](implicit ev: A <:< (T, U)): immutable.Map[T, U] =
+    immutable.Map.from(toIterable.asInstanceOf[Iterable[(T, U)]])
+
+  def toSet[B >: A]: immutable.Set[B] = immutable.Set.from(toIterable)
+
+  def toSeq[B >: A]: immutable.Seq[B] = immutable.Seq.from(toIterable)
```

Comment:
Is it useful to have this `B` type parameter? Why not just return an `immutable.Seq[A]`?

Reply:
I thought it's currently done this way (for some reason) but it turns out it is not.